tests/test_snps.py

```diff
@@ -131,6 +131,8 @@ def test_build_detected_PAR_snps(self):
             snps = SNPs("tests/input/GRCh37_PAR.csv")
             assert snps.build == 37
             assert snps.build_detected
+            # rs113378274 was merged into rs72608386 and appears on the X chromosome
+            assert snps.snps.loc["rs113378274", "chrom"] == "X"
```

Comment:
This seems to be the wrong place for this.  Would you prefer a different test? or is the test below: test_remap_snps_37_to_38_with_PAR_SNP sufficient?

Reply:
I deleted this - as this test is responsible for testing the build detection.  If you would like another  test, let me know.

Reply:
`test_remap_snps_37_to_38_with_PAR_SNP` is sufficient since it tests the end result.